minimint/src/consensus/mod.rs

```diff
@@ -513,6 +514,19 @@ where
     fn build_interconnect(&self) -> MinimintInterconnect<R> {
         MinimintInterconnect { minimint: self }
     }
+
+    pub fn list_gateways(&self) -> Vec<LightningGateway> {
+        self.db
+            .find_by_prefix(&LightningGatewayKeyPrefix)
+            .map(|res| res.expect("DB error").1)
+            .collect()
+    }
+
+    pub fn register_gateway(&self, gateway: LightningGateway) {
+        self.db
+            .insert_entry(&LightningGatewayKey(gateway.node_pub_key), &gateway)
+            .expect("DB error");
+    }
```

Comment:
What's the reasoning behind putting these endpoints into the federation consensus itself instead of the LN module?

Reply:
That was a mistake. Fixed now.